addons/l10n_latam_check/i18n/es.po

```diff
@@ -0,0 +1,535 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* l10n_latam_check
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 15.5alpha1+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-07-29 22:05+0000\n"
+"PO-Revision-Date: 2022-07-29 22:05+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: l10n_latam_check
+#: code:addons/l10n_latam_check/models/l10n_latam_checkbook.py:0
+#, python-format
+msgid " up to %s"
+msgstr "hasta %s"
```

Comment:
@jjscarafia I noticed an error here, can you help me add a "space" at the beginning before "hasta %s"
now is "hasta %s"
It should be " hasta %s"
thanks!

Reply:
@jjscarafia just not to forget about this minor change.
Can you help me with this?
Thanks!
Cheers!

Reply:
fixed